utest/shim.h

```diff
@@ -59,7 +59,7 @@
 #       define UTEST_SHIM_SCHEDULER_USE_MINAR UTEST_MINAR_AVAILABLE
 #   endif
 #   ifndef UTEST_SHIM_SCHEDULER_USE_US_TICKER
-#       define UTEST_SHIM_SCHEDULER_USE_US_TICKER (!UTEST_MINAR_AVAILABLE)
+#       define UTEST_SHIM_SCHEDULER_USE_US_TICKER 0
```

Comment:
I added you as a collaborator, so you can make the changes faster, as I cannot test this right now?